src/recipes/recipes.tsx

```diff
@@ -38,17 +37,17 @@ export const AllRecipes: React.FunctionComponent = props => {
             <Grid container className={classes.root} spacing={2}>
                 <Grid item xs={12}>
                     <Grid container justify="center" spacing={2}>
-                        {recipes.map((allRecipes, id: number) => (
-                            <Grid key={id} item>
+                        {recipes.map((recipe, index: number) => (
+                            <Grid key={index} item>
```

Comment:
Don't use `index` as the `key`

Instead use `recipe.id`

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318